utils/renderTestFiles.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash -x
+
```

Comment:
nit: make directories be based off the relative directory of script, using path relative to ` $(dirname "$0")`